hack/generator/pkg/astmodel/kubernetes_resource_interface.go

```diff
@@ -66,6 +70,11 @@ type objectFunction struct {
 
 var _ Function = &objectFunction{}
 
+// Name returns the unique name of this function
```

Comment:
Is it unique?

Reply:
Agree with @babbageclunk (sorta same comment I made above)

Reply:
I believe comments should add information that's not discernable by reading the code, so I've extended the comment to explain how it is unique.